### DISCUSSION TITLE
Thread pool performance - reddit post

### DISCUSSION THREAD

**Discussion by Disservin** [ptsouchlos/thread-pool#56] [General]
I thought you might be interested so here’s a reddit post which discusses some aspects of your library ;)
https://www.reddit.com/r/cpp/s/vA1PtWMfAV

**Comment by ptsouchlos** [comment]
Thanks for posting this @Disservin! Interesting findings and I appreciate the candid feedback. Some of the comments seem a bit harsh but overall I agree with your assessment of my library. I was already aware of the limitations you pointed out based on feedback I got from one of my previous reddit [posts](https://www.reddit.com/r/Cplusplus/comments/15h60k7/version_062_of_dpthread_pool_a_fast_c20_work/). 

I've slowly been working on a lock free work stealing deque based on the Chase Lev paper, but it's not really ready for prime time (see https://github.com/DeveloperPaul123/thread-pool/tree/feature/lock-free-deque). I do eventually want to implement a SPMC deque/queue to address the needs of my thread pool. When that time comes, I'd love to do another comparison to your new library. Thanks again!

**Reply by Disservin** [reply]
> your assessment of my library. 

not my assessment ; ) I just came across this post and thought I might share it.

**Reply by ptsouchlos** [reply]
Ah ok, thanks for clarifying!
